src/include/daos_kv.h

```diff
@@ -79,7 +79,7 @@ daos_kv_put(daos_handle_t oh, daos_handle_t th, uint64_t flags, const char *key,
  * \param[in,out]
  *		size	[in]: Size of the user buf. if the size is unknown, set
  *			to DAOS_REC_ANY). [out]: The actual size of the value.
- * \param[in]	buf	Pointer to user buffer. If NULL, only size is returned.
+ * \param[out]	buf	Pointer to user buffer. If NULL, only size is returned.
```

Comment:
should be in,out then?

Reply:
Good point.  I was thinking out as the contents of the buffer is passed out, but perhaps it should be in because the pointer itself is in.  How is this handled normally?